src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -156,14 +161,18 @@ std::unique_ptr<model_spec> drawing_classifier::init_model() const {
       ss.str("");
       ss << prefix << "_pool" << ii << _suffix;
       output_name = ss.str();
+
+      auto pad_type =padding_type::SAME;
+      if (ii == 2) pad_type = padding_type::VALID;
+
       result->add_pooling(
           /* name                 */ output_name,
           /* input                */ input_name,
           /* kernel_height        */ 2,
           /* kernel_width         */ 2,
           /* stride_height        */ 2,
           /* stride_width         */ 2,
-          /* padding              */ padding_type::VALID,
```

Comment:
align with TF implemeation

Reply:
What does the exported CoreML do here in 5.8? We want to align with that

Reply:
let me check.

Reply:
all paddings of pooling are using 'VALID'

Reply:
@jakesabathia2 I told you ;-)